gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -121,6 +189,26 @@ def configure_interfaces(self, context, resource_data):
 
         """
 
+        mgmt_ip = resource_data['mgmt_ip']
+
+        try:
+            result_log_forward = self._configure_log_forwarding(
+                const.request_url, mgmt_ip, const.CONFIGURATION_SERVER_PORT)
+        except Exception as err:
+            msg = ("Failed to configure log forwarding to Visibility VM. "
```

Comment:
Lets not talk about visibility VM in any of the logs. We should only mention about log forwarding.